test/service/test_request_validation.py

```diff
@@ -265,3 +271,21 @@ def test(url, message, params):
             test(url,
                  params={},
                  message='Missing required query parameters `entity_type`, `integration_type`')
+
+
+class CatalogNameValidationTest(WebServiceTestCase):
+
+    def test_bad_catalog_param(self):
+        url = self.base_url
+        path = '/index/files'
+        for invalid_catalog_name, error_message in [
+            ('banana', f"BadRequestError: Catalog name 'banana' is invalid must be within {[*config.catalogs]!r}"),
+            ('wassup dawg', "BadRequestError: Catalog name 'wassup dawg' contains invalid characters.")
```

Comment:
```suggestion
            ('foo bar', "BadRequestError: Catalog name 'foo bar' contains invalid characters.")
```

I'd like to avoid vernacular.